.github/workflows/checks.yml

```diff
@@ -26,3 +26,25 @@ jobs:
         uses: andresz1/size-limit-action@master
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
+  typings:
```

Comment:
please add check for storybook https://github.com/reactchartjs/react-chartjs-2/blob/master/.github/workflows/checks.yml#L48

Reply:
added